CMakeLists.txt

```diff
@@ -101,7 +101,9 @@ find_python_module(yaml 3.12 REQUIRED)
 include_directories(${NMODL_PROJECT_SOURCE_DIR}
                     ${NMODL_PROJECT_SOURCE_DIR}/src
                     ${PROJECT_BINARY_DIR}/src
-                    ${NMODL_PROJECT_SOURCE_DIR}/ext)
+                    ${NMODL_PROJECT_SOURCE_DIR}/ext
+                    ${NMODL_PROJECT_SOURCE_DIR}/ext/fmt/include
```

Comment:
We use `add_subdirectory(fmt)` for our `fmt` submodule - I wonder if that's easier than the old `include_directories` way.

Reply:
but for include to work, that is needed. Isn't it?

Reply:
`fmt` has a bunch of stuff (https://github.com/fmtlib/fmt/blob/master/CMakeLists.txt#L10) that does the heavy lifting, so then you don't have to use the old cmake style include_directories (which is nice, because then you can control warnings and such.

Reply:
Hmm.. I have `add_subdirectory(ext/fmt EXCLUDE_FROM_ALL)` to exclude it from `make/make install` and then I am getting:

```
/gpfs/bbp.cscs.ch/project/proj16/kumbhar/pramod_scratch/tmp/nmodl/src/codegen/codegen_acc_visitor.cpp:8:10: fatal error: fmt/format.h: No such file or directory
 #include "fmt/format.h"
          ^~~~~~~~~~~~~~
```

So for now I will continue using include path.

Reply:
By the way, I will go ahead with the merge as we need this sooner. But if you have more suggestions, please post the review when time permits and I will accommodate that.

Reply:
The include path is basically like a global variable, with all the problems that brings.  The problem you're having relates to not specifying the dependency for `codegen`.

I have an example PR on how to clean this up: https://github.com/BlueBrain/nmodl/pull/273

I don't know your policies well enough to shepherd it through, so I will leave it up to you on how to handle it - ie: you're welcome to take it over.